.github/ISSUE_TEMPLATE/image-provider-api-integration-request.md

```diff
@@ -1,50 +1,51 @@
 ---
-name: Image Provider API Integration Request
+name: Image provider API integration request
 about: Tell us about an API providing CC-licensed images
-title: "[API Integration] Provider name here"
-labels: enhancement, awaiting triage, ticket work required, providers
-assignees: ''
-
+labels: "🛠 goal: addition, 🚦 status: awaiting triage"
+title: "[API integration] <Replace this with actual title>"
 ---
 
-## Provider API Endpoint / Documentation
-<!-- Please provide links to the API endpoint, and any known documentation -->
+## Description
+<!-- Concisely describe the image provider. -->
 
-## Provider description
-<!-- Please provide a clear and concise description of the image provider -->
+## API endpoint / documentation
+<!-- Provide links to the API endpoint, and associated documentation. -->
 
-## Licenses Provided
-<!-- Which licenses does the provider use for images (if known) -->
+## Licenses provided
+<!-- Which CC licenses or Public Domain tools are in use by the source, if known? -->
 
-## Provider API Technical info
+## Technical details
 <!-- Please provide any technical details that might be useful for -->
 <!-- implementation, e.g., rate limits, filtering options, overall volume, -->
 <!-- etc. -->
 
-## Checklist to complete before beginning development
-<!-- please do not modify this section -->
+## Checklist
+<!-- Do not modify this section. -->
 
-No development should be done on a Provider API Script until the following info is gathered:
+No development should be done on a provider API script until the following info is gathered.
 
+<!-- Replace  the [ ] with [x] to check the boxes. --> 
 - [ ] Verify there is a way to retrieve the entire relevant portion of the provider's collection in a systematic way via their API.
-- [ ] Verify the API provides license info (license type and version; license URL provides both, and is preferred)
+- [ ] Verify the API provides license info viz. license type and version (license URL provides both, and is preferred).
 - [ ] Verify the API provides stable direct links to individual works.
 - [ ] Verify the API provides a stable landing page URL to individual works.
 - [ ] Note other info the API provides, such as thumbnails, dimensions, attribution info (required if non-CC0 licenses will be kept), title, description, other meta data, tags, etc.
 - [ ] Attach example responses to API queries that have the relevant info.
 
-## General Recommendations for implementation
-<!-- modify this section if necessary -->
+## Implementation guidelines
```

Comment:
I would highly recommend moving this into a document like `CONTRIBUTING.md` rather than having it in the issue template.

Reply:
Resolved in d9153c2.